scapy/config.py

```diff
@@ -326,26 +326,20 @@ def isCryptographyAdvanced():
     else:
         return True
 
-
-def _prompt_changer(attr,val):
-    prompt = conf.prompt
+def _prompt_changer(attr, val):
```

Comment:
Could you add the following docstring:
""Change the current prompt theme"""

Reply:
Done